admin-ui/app/company/CompanyStyle.tsx

```diff
@@ -0,0 +1,59 @@
+const CompanyStyle = {
+  errormessage: {
```

Comment:
this should be common file error, label etc, even component must have to handle this